README.md

```diff
@@ -1,43 +1,85 @@
+- A [generator][generate_usage] that invokes your protoc plugins based on a configurable [template][templates].
+  A [protoc replacement][protoc] that uses Buf's [high-performance Protobuf compiler][compiler].
+- A configurable file [builder][build_usage] that produces [Images], our extension of Protobuf's native [FileDescriptorSets][filedescriptorset].
+
+See more [features] below.
+
+## Installation
+
+### Homebrew
 
-Buf’s long-term goal is to enable schema-driven development: a future where APIs are defined consistently, in a way that service owners and clients can depend on.
+You can install `buf` using [Homebrew][brew] (macOS or Linux) through the [`bufbuild/bufbrew`][tap] tap.
 
-Defining APIs using an [IDL](https://en.wikipedia.org/wiki/Interface_description_language) provides a number of benefits over simply exposing JSON/REST services, and today, [Protobuf](https://developers.google.com/protocol-buffers) is the most stable, widely-adopted IDL in the industry.
+```sh
+brew tap bufbuild/bufbrew install buf
+brew install buf
+```
 
-However, as it stands, using Protobuf is much more difficult than using JSON as your data transfer format.
+This installs:
 
-Enter Buf: We’re building tooling to make Protobuf reliable and easy to use for service owners and clients, while keeping it the obvious choice on the technical merits.
+* The `buf`, [`protoc-gen-buf-breaking`][breaking], and [`protoc-gen-buf-lint`][lint] binaries
+* Shell completion scripts for [Bash], [Fish], [Powershell], and [zsh]
 
-Your organization should not have to reinvent the wheel to create, maintain, and consume Protobuf APIs efficiently and effectively. We'll handle your Protobuf management strategy for you, so you can focus on what matters.
+### Other methods
 
-We’re working quickly to build a modern Protobuf ecosystem. Our first tool is the **Buf CLI**, built to help you create consistent Protobuf APIs that preserve compatibility and comply with design best-practices. The tool is currently available on an open-source basis.
+For other installation methods, see our [official documentation][install], which covers:
 
-Our second tool, the **Buf Schema Registry (“BSR”)**, will be the hub of our ecosystem. The BSR is a platform that serves as the source of truth for your organization's Protobuf files, enabling you to centrally maintain compatibility and manage dependencies, while enabling your clients to consume APIs reliably and efficiently. The BSR will be available for a limited, free private beta shortly.
+* Installing `buf` on [Windows]
+* Using `buf` as a [Docker image][docker]
+* Installing as a [binary], from a [tarball], and from [source] through [GitHub Releases][releases]
+* [Verifying] releases using a [minisign] public key
+
+## Commands
+
+The best way to learn about `buf` commands and flags is through the help interface:
+
+```sh
+buf --help
+```
+
+## Goal
```

Comment:
I was envisioning that this README would get much simpler - I can't quite tell, but I think it might be longer than the original? I didn't think we'd have any of the below context, or at least only parts of it.

Reply:
I meant to split this up into multiple PRs but I'll just go ahead and do the whole thing